lib/src/components/NxCodeSnippet/NxCodeSnippet.scss

```diff
@@ -44,7 +44,7 @@
     width: 100%;
 
     .nx-text-input__box {
-      border-color: var(--nx-color-form-element-border);
+      border-color: var(--nx-swatch-grey-05);
```

Comment:
@alexalyons126 is this actually what we want? It matches kaleido but effectively what this means is that the text field in an `NxCodeSnippet` would always look the way a _hovered_ text input is supposed to.

Reply:
> > @alexalyons126 is this actually what we want? It matches kaleido but effectively what this means is that the text field in an `NxCodeSnippet` would always look the way a _hovered_ text input is supposed to.
> 
> I reached out to @alexalyons126 on Slack to confirm.

Confirmed with @alexalyons126 that this should match other form fields.
Reverted here:
4c97190c807354f3883f3f2691fe6a38a1f3c2e7